### PR TITLE
Update CSS token per new design system theming capability

### PR DESCRIPTION
**Pull request by fajar-apri-alaska**
# Alaska Airlines Pull Request

**Fixes:** #183

## Summary:

- Update the design tokens to reflect the new theming conventions regarding tokens from the AuroDesignTokens and JetstreamDesignTokens repo.

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/style.scss [hunk 1]
```diff
@@ -82,6 +83,7 @@
 
     &--ondark {
       &:hover {
+        // This somehow doesn't refer to anywhere
         color: var(--auro-hyperlink-ondark-hover-color);
```
Comment (jason-capsule42) [inline src/style.scss:87]: This new `--ds-` token doesn't exist.

We should figure out what the old token resolved to for the value and then find a token in the new DS tokens with the same value.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/style.scss (hunk 1 above)
Comment (fajar-apri-alaska) [reply src/style.scss:87]: Agree, and I'd love to do that. Problem is I couldn't find the old reference for `--auro-hyperlink-ondark-hover-color`.
Not sure if this token name refers to this [figma](https://www.figma.com/file/TSFOFDwWQxmpPpYQ2F06lH/Hyperlink?type=design&node-id=322-813&mode=design&t=IXUlR4ZVKsSZPSZS-0), or there is custom token that specifically created for hyperlink only.

image

If it is, then maybe the real token name is `--auro-color-text-primary-ondark` or `--ds-color-text-primary-default` based on the blueprint.

Please advise, thank you.

Comment (jason-capsule42) [reply src/style.scss:87]: @leeejune can you help us figure out what token should be used here?

Comment (leeejune) [reply src/style.scss:87]: The real token name for `--auro-color-text-primary-on-light` is `--ds-color-text-primary-default`. The `-on-light` has been converted to `-default` and `-on-dark` is now `-inverse`

Comment (jason-capsule42) [reply src/style.scss:87]: @fajar-apri-alaska talked with @leeejune directly and it seems we should go ahead and use `--auro-color-ui-hover-on-dark` here.

File: package.json [hunk 2]
```diff
@@ -30,8 +30,9 @@
   "devDependencies": {
     "@alaskaairux/design-tokens": "^3.15.5",
     "@alaskaairux/icons": "^4.28.0",
+    "@aurodesignsystem/design-tokens": "^4.1.1",
```
Comment (blackfalcon) [inline package.json:33]: Remove legacy version and update peerDepedencies

File: package.json [hunk 3]
```diff
@@ -30,8 +30,9 @@
...
     "@aurodesignsystem/eslint-config": "^1.3.0",
-    "@aurodesignsystem/webcorestylesheets": "^4.8.0",
+    "@aurodesignsystem/webcorestylesheets": "^5.0.2",
```
Comment (blackfalcon) [inline package.json:35]: Remove legacy version and update peerDepedencies

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -23,15 +23,15 @@
     "lit": "^2.8.0"
   },
   "peerDependencies": {
-    "@alaskaairux/design-tokens": "^3.15.5",
     "@alaskaairux/icons": "^4.20.0",
-    "@aurodesignsystem/webcorestylesheets": "^4.8.0"
+    "@aurodesignsystem/design-tokens": "^4.1.1",
```
Comment (jason-capsule42) [inline package.json:27]: Per recent discussions on the Auro team we need to keep around the legacy versions as well for the time being.
```suggestion
    "@alaskaairux/design-tokens": "^3.15.5",
    "@aurodesignsystem/design-tokens": "^4.1.1",
```

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

**Comment by fajar-apri-alaska** [comment]
I have pushed a new commit per your feedback @jason-capsule42. Thanks.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**